src/main/java/frc/robot/constants/ConstMotion.java

```diff
@@ -21,9 +26,23 @@ public class ConstMotion {
 
   public static final TalonFXConfiguration HOOD_CONFIGURATION = new TalonFXConfiguration();
 
+  public static final TalonFXConfiguration INTAKE_PIVOT_CONFIGURATION = new TalonFXConfiguration();
+
+  public static final Angle PANEL_ANGLE = Degrees.of(0);
+
   static {
 
     HOOD_CONFIGURATION.MotorOutput.NeutralMode = NeutralModeValue.Brake;
     HOOD_CONFIGURATION.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+
+    INTAKE_PIVOT_CONFIGURATION.Slot0.kS = 0;
+    INTAKE_PIVOT_CONFIGURATION.Slot0.kG = 0;
+
+    INTAKE_PIVOT_CONFIGURATION.Slot0.kP = 0;
+
+    INTAKE_PIVOT_CONFIGURATION.Slot0.GravityType = GravityTypeValue.Arm_Cosine;
+
+    INTAKE_PIVOT_CONFIGURATION.Feedback.SensorToMechanismRatio = ((0 / 0) * (0 / 0));
```

Comment:
Code is good but we but placeholder should be 1 so we dont get divide by zero error
```suggestion
    INTAKE_PIVOT_CONFIGURATION.Feedback.SensorToMechanismRatio = 1; // TODO: get value from mechanical